my-page-app/src/components/hyttebooking/MonthOverview.tsx

```diff
@@ -91,11 +91,19 @@ export default function MonthOverview() {
     },
   })
 
+  const [isDateValidForReservation, setIsDateValidForReservation] =
+    useState(true)
```

Comment:
Enda bedre:
 ```suggestion
  const isDateValidForReservation = isBefore(date, new Date(cutOffDateVacancies))
```

Sliter litt å se hvor `cutOffDateVacancies` finnes i denne filen, er den importert/sent som en prop noe sted? Så vidt jeg kan se blir den enten `undefined`, eller så kræsjer koden, ved første øyekast her på github 🤔 

Trenger ikke dytte denne inn i en egen state.